build.sbt

```diff
@@ -46,3 +54,8 @@ lazy val it = project
     Test / scalafmtOnCompile := true,
     Test / unmanagedResourceDirectories += baseDirectory.value / "it" / "test" / "resources"
   )
+  .settings(inConfig(Test)(testSettings) *)
+  .settings(resolvers += Resolver.jcenterRepo)
+  .settings(CodeCoverageSettings.settings *)
+
+addCommandAlias("testc", "; clean ; coverage ; test ; coverageReport ;")
```

Comment:
I tried to add it:tests here to include them in the coverage run which requires switching sbt projects part way through but unfortunately it doesn't persist the coverage window - if anyone knows how to now run it:tests with our unit tests for coverage please let me know

Reply:
`  .settings(inConfig(Test)(testSettings) *)`
do we need this line ? 49-54 are same. Should we use 49-54 only or reuse testSettings only ?

Reply:
addCommandAlias("testc", "; clean ; coverage ; test ; it:test ; coverageReport")
Is this not working as expected ?

Reply:
We have already a file (runtests.sh) under the root directory. Is this not helping us enough to see the coverage ?

Reply:
I'm not a big fan of using that file because we can't tailor the sbt run settings on the fly (more sbt process memory for instance)